src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -841,16 +842,33 @@ public async Task<IList<Check>> GetPullRequestChecksAsync(string pullRequestUrl)
         }
 
         /// <summary>
-        ///     Retrieve the list of reviews on a PR
+        ///  Retrieve the list of all relevant reviews on a PR. This is defined as
+        ///   - Not a comment; comments are not reviews, may be created after an approval / rejection, and may be created by the author
+        ///   - Latest response by that user; all other responses are considered valid and we'll inspect the most recent one.
+        ///     (this allows users to reject, then later approve, a PR)
         /// </summary>
         /// <param name="pullRequestUrl">Uri of pull request</param>
         /// <returns>List of reviews.</returns>
-        public async Task<IList<Review>> GetPullRequestReviewsAsync(string pullRequestUrl)
+        public async Task<IList<Review>> GetLatestPullRequestReviewsAsync(string pullRequestUrl)
         {
             (string owner, string repo, int id) = ParsePullRequestUri(pullRequestUrl);
 
             var reviews = await Client.Repository.PullRequest.Review.GetAll(owner, repo, id);
-            return reviews.Select(review =>
+
+            // Filter out comments because they could come after Approved/ChangedRequested, and they don't change the decision.
```

Comment:
[nit]
```suggestion
            // Filter out comments because they could come after Approved/ChangesRequested, and they don't change the decision.
```

Reply:
oops noticed this too late.  I fixed at least one typo though.